### ISSUE TITLE
[Feature Request] Set different speed requirements for composite characters

### ISSUE THREAD

**Issue by mgtaboada** [aradzie/keybr.com#181]
It may be a skill issue on my side, but i'm typing in Spanish and, even though my speed is kind of close to my objective, it is hard to keep up with the objective when typing characters that need a sequence of keys rather than one key or two keys at the same time. For example, for typing "á" we need to press the accent key and then the "a" key, so the speed at which we can type it is around half of the speed for typing only "a". I would like an option to configure a different speed objective for such characters.

**Comment by aradzie** [comment]
Yes, it is a know issue. I promise to do something about it in the coming days.

**Comment by guspimentel** [comment]
Hey, @aradzie!
It'd be awesome to have a solution for this issue!
